site-root/sass/skeleton/modules/_panels.scss

```diff
@@ -0,0 +1,65 @@
+.panel {
```

Comment:
@jmealo I pushed ryon's changes here upstream to skeleton-v2 and pulled them down to cbl.v1.slate.is. You can delete your override of this file and remove it from the PR. Use the EmergencePullTool to pull the new version down into your instance from cbl.v1.slate.is after you delete the override